docs/source/changes.md

```diff
@@ -124,6 +124,16 @@ authors:
 - The new [`epochs_metadata_query][config.epochs_metadata_query] setting
   allows to select epochs based on metadata query strings.
   ({{ gh(495) }} by {{ authors.hoechenberger }} and {{ authors.agramfort }})
+- The new [`time_frequency_cycles`][config.time_frequency_cycles] setting
```

Comment:
I needed this setting so I could get our TFR step to work with the ds000247 dataset; our default value leads to errors there (signal too short)